README.md

```diff
@@ -7,12 +7,12 @@
 [![Download Stats](https://img.shields.io/pypi/dm/poetry)](https://pypistats.org/packages/poetry)
 [![Discord](https://img.shields.io/discord/487711540787675139?logo=discord)][Discord]
 
-Poetry helps you declare, manage and install dependencies of Python projects,
+Poetry helps you declare, manage, and install dependencies of Python projects,
```

Comment:
I do not care to get into it myself, but you should know that if you go around submitting merge requests inserting the so-called Oxford comma, you risk getting into tedious debates: not everyone will agree that this is an improvement.